getting_started/step_by_step/ui_game_user_interface.rst

```diff
@@ -614,7 +614,7 @@ small as possible vertically. Now you can see how the interface looks in
 the context of the game.
 
 Congratulations for getting to the end of this long tutorial. You can
-find final project `here <#>`__.
+find the final project `here <#>`__.
```

Comment:
Someone should link this to `getting_started/step_by_step/files/ui_gui_design.zip`.

Reply:
Fixed.